pipelines/rj_smas__api_datametrica_agendamentos/flow.py

```
@@ -110,3 +111,106 @@ def rj_smas__api_datametrica_agendamentos(
+        "filter_dispatched_phones_or_cpfs": None,
+        "filter_duplicated_phones": False,
+        "filter_duplicated_cpfs": True,
+        "whitelist_percentage": 0,
+        "infisical_secret_path": "/crm_disparo_template",
+        "whitelist_environment": "production",
+        "query": """WITH
+                    segmentacao_original AS (
+                        SELECT
+                            LPAD(CAST(cpf AS STRING), 11, '0') AS cpf,
+                            `rj-crm-registry.udf.VALIDATE_AND_FORMAT_CELLPHONE`(
+                                REGEXP_REPLACE(telefone, r'[^\d]', '')
```

Ruff (0.15.21)

[warning] Invalid escape sequence: `\d`

Use a raw string literal

(W605)